docs/articles/documentation/using-testcafe/common-concepts/live-mode.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: docs
+title: Live Mode
+permalink: /documentation/using-testcafe/common-concepts/live-mode.html
+---
+# Live Mode
+
+Live mode keeps the TestCafe process and browsers opened the whole time you are working on tests. Changes you make in code immediately restart the tests. That is, live mode allows you to see test results instantly.
```

Comment:
Live mode ensures the TestCafe process and browsers remain active while you work on tests. You can see test results instantly because the tests are restarted when you make changes.